src/fsharp/FSharp.Compiler.Service.netcore/project.json

```diff
@@ -0,0 +1,298 @@
+    "../../ilx/EraseClosures.fs",
+    "../../ilx/EraseUnions.fsi",
+    "../../ilx/EraseUnions.fs",
+    "../UnicodeLexing.fsi",
+    "../UnicodeLexing.fs",
+    "../layout.fsi",
+    "../layout.fs",
+    "../ast.fs",
+    "pppars.fs",
+    "pars.fs",
+    "../lexhelp.fsi",
+    "../lexhelp.fs",
+    "pplex.fs",
+    "lex.fs",
+    "../LexFilter.fs",
+    "../tainted.fsi",
+    "../tainted.fs",
+    "../ExtensionTyping.fsi",
+    "../ExtensionTyping.fs",
+    "../QuotationPickler.fsi",
+    "../QuotationPickler.fs",
+    "../tast.fs",
+    "../TcGlobals.fs",
+    "../TastOps.fsi",
+    "../TastOps.fs",
+    "../TastPickle.fsi",
+    "../TastPickle.fs",
+    "../import.fsi",
+    "../import.fs",
+    "../infos.fs",
+    "../NicePrint.fs",
+    "../AugmentWithHashCompare.fsi",
+    "../AugmentWithHashCompare.fs",
+    "../NameResolution.fsi",
+    "../NameResolution.fs",
+    "../TypeRelations.fs",
+    "../PatternMatchCompilation.fsi",
+    "../PatternMatchCompilation.fs",
+    "../ConstraintSolver.fsi",
+    "../ConstraintSolver.fs",
+    "../CheckFormatStrings.fsi",
+    "../CheckFormatStrings.fs",
+    "../FindUnsolved.fs",
+    "../QuotationTranslator.fsi",
+    "../QuotationTranslator.fs",
+    "../PostInferenceChecks.fsi",
+    "../PostInferenceChecks.fs",
+    "../TypeChecker.fsi",
+    "../TypeChecker.fs",
+    "../Optimizer.fsi",
+    "../Optimizer.fs",
+    "../DetupleArgs.fsi",
+    "../DetupleArgs.fs",
+    "../InnerLambdasToTopLevelFuncs.fsi",
+    "../InnerLambdasToTopLevelFuncs.fs",
+    "../LowerCallsAndSeqs.fs",
+    "../autobox.fs",
+    "../IlxGen.fsi",
+    "../IlxGen.fs",
+    "../CompileOps.fsi",
+    "../CompileOps.fs",
+    "../CompileOptions.fsi",
+    "../CompileOptions.fs",
+    "../fsc.fsi",
+    "../fsc.fs",
+    "../vs/IncrementalBuild.fsi",
+    "../vs/IncrementalBuild.fs",
+    "../vs/Reactor.fsi",
+    "../vs/Reactor.fs",
+    "../vs/ServiceConstants.fs",
+    "../vs/ServiceDeclarations.fsi",
+    "../vs/ServiceDeclarations.fs",
+    "../vs/Symbols.fsi",
+    "../vs/Symbols.fs",
+    "../vs/Exprs.fsi",
+    "../vs/Exprs.fs",
+    "../vs/ServiceLexing.fsi",
+    "../vs/ServiceLexing.fs",
+    "../vs/ServiceParseTreeWalk.fs",
+    "../vs/ServiceNavigation.fsi",
+    "../vs/ServiceNavigation.fs",
+    "../vs/ServiceParamInfoLocations.fsi",
+    "../vs/ServiceParamInfoLocations.fs",
+    "../vs/ServiceUntypedParse.fsi",
+    "../vs/ServiceUntypedParse.fs",
+    "../vs/service.fsi",
+    "../vs/service.fs",
+    "../vs/SimpleServices.fsi",
+    "../vs/SimpleServices.fs",
+    "../fsi/fsi.fsi",
+    "../fsi/fsi.fs"
+  ],
+  "dependencies": {
+    "Microsoft.FSharp.Core.netcore": "1.0.0-*",
+    "Microsoft.NETCore.App": {
+      "type": "platform",
+      "version": "1.0.0-rc2-3002426"
+    }
+  },
+  "tools": {
+    "dotnet-fssrgen": {
+      "version": "2.0.1",
+      "imports": [
+        "dnxcore50",
+        "portable-net45+win81",
+        "netstandard1.3"
+      ]
+    },
+    "dotnet-compile-fsc": {
```

Comment:
that should not be there, are you using .net cli version `1.0.0-beta-002071`? if not pls use that version